LOAN/LoanTypes/LoanProducts.rdf

```diff
@@ -52,35 +52,66 @@
 		<rdfs:subClassOf>
 			<owl:Restriction>
 				<owl:onProperty rdf:resource="https://spec.edmcouncil.org/fibo/ontology/FBC/ProductsAndServices/FinancialProductsAndServices/isEmbodiedIn"/>
-				<owl:someValuesFrom rdf:resource="&fibo-loan-ln-ln;LoanContract"/>
+				<owl:someValuesFrom rdf:resource="&fibo-loan-ln-ln;Loan"/>
 			</owl:Restriction>
 		</rdfs:subClassOf>
 		<rdfs:subClassOf>
 			<owl:Restriction>
-				<owl:onProperty rdf:resource="&fibo-loan-typ-prod;hasPreconditions"/>
-				<owl:someValuesFrom rdf:resource="https://spec.edmcouncil.org/fibo/ontology/FBC/DebtAndEquities/CreditFacilities/LoanPrecedentConditions"/>
+				<owl:onProperty rdf:resource="&fibo-fnd-rel-rel;refersTo"/>
+				<owl:someValuesFrom rdf:resource="&fibo-fbc-dae-dbt;Collateral"/>
 			</owl:Restriction>
 		</rdfs:subClassOf>
 		<rdfs:subClassOf>
 			<owl:Restriction>
-				<owl:onProperty rdf:resource="&fibo-loan-typ-prod;isTemplateFor"/>
-				<owl:someValuesFrom rdf:resource="&fibo-loan-ln-ln;LoanContract"/>
+				<owl:onProperty rdf:resource="&fibo-loan-typ-prod;hasPreconditions"/>
+				<owl:someValuesFrom rdf:resource="https://spec.edmcouncil.org/fibo/ontology/FBC/DebtAndEquities/CreditFacilities/LoanPrecedentConditions"/>
 			</owl:Restriction>
 		</rdfs:subClassOf>
 		<rdfs:subClassOf>
 			<owl:Restriction>
-				<owl:onProperty rdf:resource="&fibo-loan-typ-prod;specifies"/>
-				<owl:someValuesFrom rdf:resource="&fibo-loan-loan-col;LoanCollateral"/>
+				<owl:onProperty rdf:resource="&fibo-loan-typ-prod;isTemplateFor"/>
+				<owl:someValuesFrom rdf:resource="&fibo-loan-ln-ln;Loan"/>
 			</owl:Restriction>
 		</rdfs:subClassOf>
 	</owl:Class>
 	
+	<owl:Class rdf:about="&fibo-loan-typ-prod;AutoLoanPurpose">
```

Comment:
Not sure these subclasses of LoanPurpose have any purpose themselves.

Reply:
I know - and they should be individuals if there is something that they do.  I moved the because they were referenced and I wanted to delete the ontology that they were in.